hs_app_timeseries/forms.py

```diff
@@ -112,22 +200,36 @@ class MethodValidationForm(forms.Form):
 
 
 class ProcessingLevelFormHelper(BaseFormHelper):
-    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None, *args, **kwargs):
-        # the order in which the model fields are listed for the FieldSet is the order these fields will be displayed
+    def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_name=None,
+                 *args, **kwargs):
+        # the order in which the model fields are listed for the FieldSet is the order these
+        # fields will be displayed
         field_width = 'form-control input-sm'
         layout = Layout(
                      Field('processing_level_code', css_class=field_width),
                      Field('definition', css_class=field_width),
                      Field('explanation', css_class=field_width),
                      )
         kwargs['element_name_label'] = 'Processing Level'
-        super(ProcessingLevelFormHelper, self).__init__(allow_edit, res_short_id, element_id, element_name, layout,  *args, **kwargs)
+        super(ProcessingLevelFormHelper, self).__init__(allow_edit, res_short_id, element_id,
+                                                        element_name, layout, *args, **kwargs)
 
 
 class ProcessingLevelForm(ModelForm):
     def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, **kwargs):
         super(ProcessingLevelForm, self).__init__(*args, **kwargs)
-        self.helper = ProcessingLevelFormHelper(allow_edit, res_short_id, element_id, element_name='ProcessingLevel')
+        self.helper = ProcessingLevelFormHelper(allow_edit, res_short_id, element_id,
+                                                element_name='ProcessingLevel')
+
+    @property
+    def form_id(self):
+        form_id = 'id_processinglevel_%s' % self.number
+        return form_id
+
+    @property
+    def form_id_button(self):
+        form_id = 'id_processinglevel_%s' % self.number
+        return "'" + form_id + "'"
```

Comment:
@pkdash OK. I see the difference, ```form_id_button``` property returns quoted form_id. It is OK then.